### PR TITLE
MCR-3053 adapt api to new changes in mycore

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MIR-3053).

### CODE REVIEW

**Comment by sebhofmann** [comment]
requires https://github.com/MyCoRe-Org/mycore/pull/2115 to build

**Comment by toKrause** [comment]
This is a duplication of existing PR https://github.com/MyCoRe-Org/mir/pull/962

File: mir-module/src/test/java/org/mycore/mir/acl/MIRACLTest.java [hunk 1]
```diff
@@ -47,7 +46,7 @@ public void disableDebug() {
     @Override
     public void setUp() throws Exception {
         super.setUp();
-        accessSystem = MCRConfiguration2.<MCRFactsAccessSystem>getInstanceOf("MCR.Access.Class")
+        accessSystem = MCRConfiguration2.getInstanceOf(MCRFactsAccessSystem.class, "MCR.Access.Class")
```
Comment (toKrause) [inline mir-module/src/test/java/org/mycore/mir/acl/MIRACLTest.java:49]: should rather use `getInstanceOfOrThrow` here, in order to obtain a more informative error message

**Review by toKrause — CHANGES_REQUESTED** [review]
... or use the other PR

**Pull request closed without merging**